content/en/docs/contribute/participating.md

```diff
@@ -122,6 +187,38 @@ For more about how to become a SIG Docs approver and the responsibilities and
 time commitment involved, see
 [Becoming a reviewer or approver](#becoming-an-approver-or-reviewer).
 
+#### Becoming an approver
+
+When you meet the
+[requirements for becoming an approver](https://github.com/kubernetes/community/blob/master/community-membership.md#approver),
+you can apply to become a SIG Docs approver. Even if you are already an approver
+in a different SIG, you still need to apply separately for SIG Docs.
+
+To apply, open a pull request to add yourself to the `approvers` section of the
```

Comment:
Same changes as per reviewers:

```
If your pull request is approved, you are now a SIG Docs approver. [K8s-ci-robot] (https://github.com/kubernetes/test-infra/tree/master/prow#bots-home) will automatically assign you to new pull requests.

If you are approved, request that a current SIG Docs approver add you to the [@kubernetes/sig-docs-maintainers](https://github.com/orgs/kubernetes/teams/sig-docs-maintainers)
GitHub team. Only members of the `kubernetes-website-admins` GitHub team can add new members to a GitHub team.
```
> If your pull request is approved, you are now a SIG Docs approver. [K8s-ci-robot](https://github.com/kubernetes/test-infra/tree/master/prow#bots-home) will automatically assign you to new pull requests.
> 
> If you are approved, request that a current SIG Docs approver add you to the [@kubernetes/sig-docs-maintainers](https://github.com/orgs/kubernetes/teams/sig-docs-maintainers)
GitHub team. Only members of the `kubernetes-website-admins` GitHub team can add new members to a GitHub team.